pkgs/tools/misc/fontforge/default.nix

```diff
@@ -10,6 +10,8 @@
 stdenv.mkDerivation rec {
   name = "fontforge-${version}";
   version = "20160404";
+  versionModtime = "1459728000"; # unix timestamp of ${version}
```

Comment:
Then again, maybe not worth the effort if this is fixed by itself in the next version bump.